wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitStatusProvider.java

```diff
@@ -1,119 +0,0 @@
-/*
```

Comment:
Don't remove this class for now, just mark as Deprecated

Reply:
Fixed